src/node_crypto.h

```diff
@@ -259,6 +267,17 @@ class SSLWrap {
                                      unsigned int inlen,
                                      void* arg);
 #endif  // OPENSSL_NPN_NEGOTIATED
+#ifdef TLSEXT_TYPE_application_layer_protocol_negotiation
+  static void GetALPNNegotiatedProto(
+    const v8::FunctionCallbackInfo<v8::Value>& args);
```

Comment:
I think this is where I thought it wouldn't compile with ALPN disabled (and it wouldn't, the declaration is inside the #ifdef block.)